nupic/research/frameworks/vernon/mixins/oml.py

```diff
@@ -266,7 +329,27 @@ def run_meta_testing_phase(self, num_classes_learned):
             acc = correct / len(self.test_train_loader.sampler.indices)
             meta_test_train_accuracies.append(acc)
 
-        return meta_test_train_accuracies, meta_test_test_accuracies
+        return meta_test_train_accuracies, meta_test_test_accuracies, lr
+
+    def get_named_test_train_params(self):
```

Comment:
There is only one variable difference between these 2 functions, can't they just be a `get_named_params` functions with an additional argument?

Reply:
I made a new function called `_get_params_by_names` that simplifies each of these to one line, as well as `get_named_fast_params`.